lib/bin/lwm2m_carrier/include/lwm2m_carrier.h

```diff
@@ -770,11 +762,28 @@ int lwm2m_carrier_velocity_set(int heading, float speed_h, float speed_v, float
  * @retval -EINVAL If at least one input argument is incorrect.
  * @retval -ENOMEM If there is not enough memory to copy the buffer contents to the resource model.
  */
-int lwm2m_carrier_app_data_send(const uint16_t *path, uint16_t path_len, const uint8_t *buffer,
-				size_t buffer_len);
+int lwm2m_carrier_app_data_set(const uint16_t *path, uint16_t path_len, const uint8_t *buffer,
+			       size_t buffer_len);
+
+/**
+ * @brief Schedule data to be sent to a LwM2M Server.
```

Comment:
```suggestion
 * @brief Schedule data to be sent to a LwM2M server.
```

Reply:
Upper case follows the LwM2M specification 🙂